packages/core/admin/admin/src/content-manager/pages/ListSettingsView/index.jsx

```diff
@@ -183,23 +171,16 @@ export const ListSettingsView = ({ layout, slug }) => {
             navigationAction={
               <Link
                 startIcon={<ArrowLeft />}
-                to={{
-                  to: `/content-manager/${kind}/${uid}`,
```

Comment:
Sorry there was some confusion, if you dont make the changes you've made and just change this property to `pathname` it should work as you expect it too?

Reply:
it works for me

Reply:
it works, I'll PR it